src/evaluators/SwitchStatement.js

```diff
@@ -90,32 +56,32 @@ function AbstractCaseBlockEvaluation(
       for (let i = 0; i < c.consequent.length && !(result instanceof Completion); ++i) {
         let node = c.consequent[i];
         let r = env.evaluateCompletion(node, strictCode);
+        invariant(!(r instanceof Reference));
 
-        // TODO understand Reference and how to deal with it better
         // TODO come up with a strategy for handling PossiblyNormalCompletion correctly
-        if (r instanceof Reference) {
-          throw ReportNoReferenceError();
-        } else if (r instanceof PossiblyNormalCompletion) {
-          throw ReportNoPossiblyNormalCompletionError();
+        if (r instanceof PossiblyNormalCompletion) {
+          let msg = "This operation is not yet supported on case blocks that result in a PossiblyNormalCompletion.";
+          realm.reportIntrospectionError(msg);
```

Comment:
You want to create a CompilerDiagnostic, for which you'll need a new error code, which implies that you'll update the Wiki with a nice explanation of how such a situation arises and how to avoid it.

Reply:
Aha. I see now. I'll go look into how to do that and update this PR appropriately.

Reply:
Ok. Can you confirm that the correct place to describe the (hopefully temporary) limitation is here: https://prepack.io/frequently-asked-questions.html?

Reply:
Ignore that, I found it!

Reply:
Ok. Created the page and uploading another diff to see if I have the error handling right.